.env.localdocker

```
@@ -0,0 +1,16 @@
+REMOTE_GATEWAY_API_BASE_DOCKER=http://remote-gateway:9090
+REMOTE_GATEWAY_WS_URL_LOCAL=ws://localhost:8081
+REMOTE_GATEWAY_WS_URL_DOCKER=ws://remote-gateway:8081
+TRUST_PROXY_HOPS=1
+# Use the real browser-facing host here. 0.0.0.0 is a bind address, not a valid browser Origin.
+# Replace LOCAL_HOST_OR_DOMAIN with the exact host/IP/domain and port your browser uses to open the app.
+RP_ID=LOCAL_HOST_OR_DOMAIN
+RP_ORIGIN=http://LOCAL_HOST_OR_DOMAIN:18112
+ALLOWED_WS_ORIGINS=http://LOCAL_HOST_OR_DOMAIN:18112
+ALLOWED_ORIGINS=http://LOCAL_HOST_OR_DOMAIN:18112
+CORS_ALLOWED_ORIGINS=http://LOCAL_HOST_OR_DOMAIN:18112
+REMOTE_GATEWAY_API_TOKEN=local-test-token
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REMOTE_GATEWAY_API_TOKEN key should go before the REMOTE_GATEWAY_WS_URL_DOCKER key

(UnorderedKey)